frontend/src/components/SettingsModal.jsx

```
@@ -299,467 +299,329 @@ const SettingsModal = ({ isOpen, onClose, onSave, activeModel }) => {
+        { id: 'data', label: 'Data Management', icon: Trash2 },
+    ];
+
     return (
         <>
             <div
-                className="fixed inset-0 z-50 flex items-center justify-center bg-black/60 backdrop-blur-sm"
+                className="fixed inset-0 z-50 flex items-center justify-center bg-black/70 backdrop-blur-sm"
                 onClick={(e) => {
                     if (e.target === e.currentTarget) onClose();
                 }}
             >
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)